terraform/lambda.tf

```diff
@@ -5,25 +5,25 @@
 # }
 
 
-resource "aws_lambda_layer_version" "wfnews_lambda_layer" {
+data "aws_lambda_layer_version" "wfnews_lambda_layer" {
```

Comment:
I tried data before and it choked until I added resources, hopefully not the case with the revisions!